scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -48,7 +48,7 @@
       </li>
     </ul>
     <span class="navbar-text">
-      <a href="{{ url_for('cases.pdf_case_report', institute_id=institute._id, case_name=case.display_name) }}"><button class="btn btn-sm btn-secondary" type="button">Download PDF</button></a>
+      <button class="navbar-brand" onclick="window.print();">Create PDF</button>
```

Comment:
Hm, I like it being very lightweight and potentially saving us a lot of hassle with the pdf generation stuff. But wasn't there reason we do it like this that pdf generation was missing from vanilla windows? In MacOS, the pdf generation from "print" is rather accessible, but on windows was previously dependent on additional programs. Seems latest versions do have a "Microsoft Print to PDF", one or two dialogs into the print menu depending on which browser you use (the system print dialog has it, not eg MS Edge print dialog). The quality is not great. Most colors from the report disappear. I suspect some of this could be fixed by adjusting html to fit their capabilities, much like other pdf generators we know and pain over. But at least some of that would be needed - the users will protest as is.

Reply:
I agree that the report shouldn't lose quality when downloaded with explorer. How did you try?

There is this simulation that you can make using safari: https://setapp.com/how-to/use-internet-explorer-for-mac

And if you export it checking the "print backgrounds" thing

<img width="436" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/a9d06949-0fc4-44f6-ba73-4c7a25bdf8b7">

It looks good: 

<img width="1055" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/d18726b6-9bb9-4575-b273-e1e12465dabb">




But of course we should test using a real explorer browser!

Reply:
The thing is not the browser, it’s the pdf generator on the OS. I’m pretty sure that IE-for-Mac uses the macOS pdf print. I tested with a (hospital) windows machine, looks the same with both Chrome and Edge with “MS Print to pdf”. I did not see any flags for “background” as in your Safari simulation. If we could find a setting like that on win (ideally without the user having to find it) maybe it would look better!

Reply:
I bet that's because people in general don't want to export background colors and figures.. 

Just of of curiosity, if you go into the explorer settings, like advanced, is there an option to on printing of background colors and images? There should be right? And how does it look like if you check that option?

Reply:
Oooh, you know what: there is an option to include "Bakgrundsgrafik" both in Chrome and Edge, which makes the printouts pretty ok. It's off by default, and hidden a few menus in, but perhaps possible to explain!